### ISSUE TITLE
Scanner exception on retrieving existing pull request info

### ISSUE THREAD

**Issue by mc1arke** [mc1arke/sonarqube-community-branch-plugin#6]
Whilst testing the fix for #4  an exception is thrown by the scanner engine when retrieving pull request info for a project containing an existing Pull Request analysis.

The stacktrace is

```
Unable to load component class org.sonar.scanner.scan.filesystem.InputComponentStore: Unable to load component interface org.sonar
.scanner.scan.branch.BranchConfiguration: Unable to load component class org.sonar.scanner.scan.branch.ProjectPullRequests: java.lang.NumberFormatException: For input string: "2019-04-04T19:44:27+0100" -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.6.0.1398:sonar (default-cli) on project play-about: Unable to load component class org.sonar.scanner.scan.filesystem.InputComponentSto
re
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
...
```

and the content of a GET request to `/api/project_pull_requests/list?project=xxx%3Aplay-about` is
```json
{"pullRequests":[{"key":"101","title":"dummybranch","branch":"dummybranch","base":"master","status":{"qualityGateStatus":"OK","bugs":0,"vulnerabilities":0,"codeSmells":0},"analysisDate":"2019-04-04T19:44:27+0100"}]}
```

**Comment by mildis** [comment]
We tried this correction and it worked. 👍 
Can be merged to master

**Comment by mc1arke** [comment]
Released in [1.0.2](https://github.com/mc1arke/sonarqube-community-branch-plugin/tree/1.0.2)
